subsys/bluetooth/controller/hal/nrf5/radio/radio.c

```diff
@@ -256,8 +256,6 @@ void radio_disable(void)
 void radio_status_reset(void)
 {
 	NRF_RADIO->EVENTS_READY = 0;
```

Comment:
@cvinayak  The commit message is really educative; I would suggest to add this as a comment, somewhere in the code, to indicate to future contributors why some events need to be cleared and some are not.